src/main/webapp/user_profile.html

```diff
@@ -14,7 +14,15 @@
       <a href="task.html" id="task">Task Page</a>
     </div>
     <div id="content">
-      <h1>Welcome, UserName</h1>
+      <h1>UserName</h1>
+      <div>Are you UserName? Re-confirm to make edits to your page</div>
+      <form>
+          <label for="email">email</label><br>
+          <input type="text" id="email" name="email" value="test@gmail.com"><br>
+          <label for="password">password:</label><br>
+          <input type="text" id="password" name="password" value="test"><br><br>
+          <input type="submit" value="Submit">
+      </form> 
```

Comment:
In the future, should this be a login button that redirects to a login page (using the authorization API)?

Reply:
I was thinking the person is already logged in (or else they would have no way to access their page) but this is just an extra measure of security (like when you have to re-enter password on a Mac to change some settings even though you are already logged in). That was just my thinking but I'm open to change!

Reply:
imo it would depend on what they're changing.  If they're changing passwords (or maybe email as well), I would require them to reenter their password (just the password though, the session should already know about the username/email).  If they're changing something else, it might be too much of a hassle to login again.

Reply:
> This page seems to be doing both "User Settings" and "User Profile/Activity". The difference is the former is private and should only be visible for the user. The later is public and could be visible for everyone. We also don't want them to accidentally change settings while viewing their active tasks here.

Would you suggest having 2 pages instead? or just separating them?

Reply:
separating into 2 pages.